### DISCUSSION TITLE
Migrate Docker installation to compile from source version

### DISCUSSION THREAD

**Discussion by blqdev** [open-webui/open-webui#4207] [General]
Greetings.

We set up a docker instance of Open WebUI. Is there a way we can move that configuration to another instance which we've set up to run from source? Like a backup/restore feature, even if it's manual?

**Comment by justinh-rahb** [comment]
```sh
docker cp <container name or id>:/app/backend/data open-webui-data
```
That will copy the contents of the persistent volume to a folder called `open-webui-data`, then copy the contents of this to the `backend/data/` folder of your source install:
```sh
git clone https://github.com/open-webui/open-webui
cp -r open-webui-data/* open-webui/backend/data/
```
Follow rest of normal instructions to run direct install from source, it should start with duplicated settings and database from your Docker install.
